test/dlc_tests/sanity/test_utility_installation.py

```diff
@@ -35,6 +31,7 @@ def test_awscli(mxnet_inference):
     test_utils.run_cmd_on_container(container_name, ctx, "aws --version")
 
 
+@pytest.mark.usefixtures("sagemaker")
```

Comment:
This test should only be ran on SM images. The packages are SM utility packages added soley for SM.
Let revamp the test to reflect the same.

Reply:
Sure - if you look below, I made sure that non sm* utility packages are installed in DIY - but if this only needs to be a requirement for sagemaker, then we can remove DIY and make this sm only

Reply:
All the below packages are SM utility packages and should exist only on SM DLC image.
```
"bokeh", "imageio", "plotly", "seaborn", "shap", "pandas", "cv2", "sagemaker"
```

Reply:
I think since these are open source packages not explicitly marked as sagemaker, we can also have these in the DIY image (minus the "sagemaker") dependency. Do you see any reason for moving them out of the DIY?